test/assertions/to-contain.spec.js

```diff
@@ -1,5 +1,10 @@
 /*global expect*/
 describe('to contain assertion', function () {
+    var expect = unexpected.clone().use(typeof weknowhow === 'undefined' ?
+        require('unexpected-magicpen') :
+        weknowhow.unexpectedMagicPen
+    );
+
```

Comment:
Sorry, I missed this comment. Ended up adding another `expectWithUnexpectedMagicPen` global instead. If you'd rather make the plugin a global and `.clone().use(...)` when needed, feel free :)